src/site/_includes/components/EventTable.js

```diff
@@ -41,17 +42,34 @@ module.exports = (event, authorsCollection) => {
     }
   }
 
+  const slugs = {};
+  const slugForTitle = (title) => {
+    // Find a slug for this title, but prevent duplicate IDs.
+    const base = slugify(title, {lower: true, strict: true, remove: /'/});
+    let id = base;
+    let suffix = 0;
+    while (id in slugs) {
+      id = base + ++suffix;
+    }
+    slugs[id] = title;
+    return id;
+  };
+
   const renderSession = ({speaker, title}) => {
     // Always pass an Array of author IDs.
     const authors = typeof speaker === 'string' ? [speaker] : speaker;
 
+    const id = slugForTitle(title);
+
     return html`
-      <tr>
-        <td class="w-event-schedule__speaker">
+      <div class="w-event-schedule__row" data-session-id=${id}>
```

Comment:
I think we may want the entire row to be clickable to open the modal.

Reply:
The challenge with this is that the author's heads and links are also link targets.

The other option is that we only make them clickable from within the modal. WDYT?